src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -126,3 +122,41 @@ WithEverythingDisplayed.args = {
         { label: "Secondary", action: () => null },
     ],
 };
+
+type ArrowPosition = "left" | "right" | "top" | "bottom";
+type WithArrowProps = TooltipProps & { arrowPosition: ArrowPosition };
+const getArrowStyle = (position: ArrowPosition) => {
+    switch (position) {
+        case "left":
+            return { top: 0, left: 0, transform: "translate3d(-6px, 26px, 0)" };
+        case "right":
+            return { top: 0, right: 0, transform: "translate3d(6px, 26px, 0)" };
+        case "top":
+            return { top: 0, left: "50%", transform: "translate3d(-50%, -6px, 0)" };
+        case "bottom":
+            return { bottom: 0, left: "50%", transform: "translate3d(-50%, 6px, 0)" };
+        default:
+            return {};
+    }
+};
+
+export const WithArrow: Story<WithArrowProps> = (args: WithArrowProps) => (
+    <div className="tw-relative tw-inline-block">
+        <TooltipComponent {...args}>
+            <TooltipArrow
+                style={getArrowStyle(args.arrowPosition)}
+                placement={args.arrowPosition}
+                headerColor={args.brightHeader}
+            />
+        </TooltipComponent>
+    </div>
+);
+WithArrow.args = {
+    arrowPosition: "left",
+};
+WithArrow.argTypes = {
+    arrowPosition: {
```

Comment:
That doesn't seem to have any effect. Maybe try removing it if it doesn't affect anything.